src/art/__init__.py

```diff
@@ -34,3 +35,10 @@
     "Trajectory",
     "TrajectoryGroup",
 ]
+
+# Avoid importing LocalAPI on client machines
+if sys.platform.startswith("linux"):
```

Comment:
This feels pretty ugly; maybe we should think harder about what should be imported from where.

Reply:
It is ugly. I think the problem might be solved by removing `LocalAPI` from the top `__init__.py` file. Just pushed that change.